accumulator_updater/programs/accumulator_updater/src/instructions/emit_inputs.rs

```diff
@@ -0,0 +1,165 @@
+use {
+    crate::{
+        accumulator_acc_seeds,
+        accumulator_acc_seeds_with_bump,
+        state::*,
+        AccumulatorUpdaterError,
+    },
+    anchor_lang::{
+        prelude::*,
+        system_program::{
+            self,
+            CreateAccount,
+        },
+    },
+};
+
+
+pub fn emit_inputs<'info>(
+    ctx: Context<'_, '_, '_, 'info, EmitInputs<'info>>,
+    base_account: Pubkey,
+    data: Vec<Vec<u8>>,
+    account_type: u32,
```

Comment:
do we need account_type at all? I don't remember exactly but i'm looking back at our previous convo and it's not mentioned there:

https://github.com/pyth-network/pyth-crosschain/pull/712#discussion_r1146392083

Reply:
I second this

Reply:
this was for flexibility if we ever wanted to include different types of accounts into the accumulator aside from the price accounts - maybe even if it was something outside of pyth (i don't have any specific use-case in mind and not sure if we would ever want product or mapping accounts but I can remove this if we really don't need this)

Reply:
will remove for now